src/auth/index.js

```diff
@@ -118,6 +118,41 @@ export default class Auth {
       .then(responseHandler);
   }
 
+  /**
+   * Exchanges a code obtained via a native social authentication solution for the user's tokens
+   *
+   * @param {Object} parameters parameters used to obtain tokens from a code
+   * @param {String} parameters.subjectToken code returned by the native social authentication solution
+   * @param {String} parameters.subjectTokenType identifier that indicates the type of `subjectToken`
+   * @param {String} parameters.userProfile optional element used for native iOS interactions for which profile updates can occur. Only for `APPLE-AUTHZ-CODE`
+   * @param {String} parameters.audience optional API audience to request
```

Comment:
```suggestion
   * @param {String} [parameters.audience] API audience to request
```

Reply:
Why use an array here?

Reply:
Fixed in b6cf3cb8110a349ce96ca14b5512c3e17a42fa57.